meeting/05-18-2018.md

```diff
@@ -0,0 +1,87 @@
+## Time
+UTC Fri 18-May-2018 17:00 (04:00 PM):
+
+Timezone | Date/Time
+-- | --
+US / Pacific  Fri 18-May-2018 09:00 (09:00 AM)
+US / Mountain Fri 18-May-2018 10:00 (10:00 AM)
+US / Central  Fri 18-May-2018 11:00 (11:00 AM)
+US / Eastern  Fri 18-May-2018 12:00 (12:00 PM)
+London  Fri 18-May-2018 17:00 (05:00 PM)
+Amsterdam Fri 18-May-2018 18:00 (06:00 PM)
+Moscow  Fri 18-May-2018 19:00 (07:00 PM)
+Chennai Fri 18-May-2018 21:30 (09:30 PM)
+Hangzhou  Sat 19-May-2018 00:00 (12:00 AM)
+Tokyo Sat 19-May-2018 01:00 (01:00 AM)
+Sydney  Sat 19-May-2018 02:00 (02:00 AM)
```

Comment:
The table is missing the `|` between each timezone and date/time so everything is being put in the Timezone column